samples/bluetooth/iso_broadcast/src/main.c

```diff
@@ -74,6 +79,8 @@ void main(void)
 	uint32_t iso_send_count = 0;
 	uint8_t iso_data[sizeof(iso_send_count)] = { 0 };
 	struct net_buf *buf;
+	size_t timeout_counter = reset_timeout_counter(big_create_param.interval,
+						       BIG_TERMINATE_TIMEOUT_US);
```

Comment:
It is confusing that global variable is used for interval, but a macro for terminate timeout. Have a define for interval, use it to assign the interval in the global `big_create_param` and use a macro to calculate the timeout_counter (instead of a function), as the values are constants in this sample.

Reply:
Sure :)